docs/testplan/MACsec-neighbor-reboot.md

```diff
@@ -0,0 +1,53 @@
+- [Overview](#overview)
+    - [Scope](#scope)
+    - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test cases](#test-cases)
+
+# Test name
+
+Macsec Neighbor Reboot
+
+## Overview
+
+The goal of this test is to verify that the macsec neighbor relationship is established after the neighbor is rebooted.
+
+### Scope
+
+The test is targeting a running SONIC system with fully functioning configuration. The purpose of the test is to test when a neighbor reboots while Macsec is Active.
```

Comment:
"The purpose of the test is to test when a neighbor reboots while Macsec is Active" does not make sense in English. Can you please rephrase it?

Reply:
I changed the wording on this.